packages/embed/e2e/applitools-utils.ts

```diff
@@ -1,6 +1,7 @@
 import { screenSizeDesktop, screenSizeMobile, Viewport } from './cypress-utils'
 
 const eyesCheck = async (name: string, { width, height }: Viewport) => {
+  cy.wait(2000)
```

Comment:
wouldn't be better to wait for the element? I believe Cypress has a 5 seconds default time

Reply:
I tried, but It looks like the wait is best solution.

Reply:
👍